src/main/java/blackjack/controller/BlackjackController.java

```diff
@@ -0,0 +1,131 @@
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private Dealer dealer;
+
+    public BlackjackController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        String[] playersName = inputView.receivePlayersName();
+        Players players = PlayersFactory.from(playersName);
+        dealer = new Dealer(players);
+        setting();
+        for (Player player : dealer.getPlayers().getPlayers()) {
+            play(player);
+        }
+        turnOfDealer();
+        finishGame();
+    }
+
+    private void setting() {
+        dealer.drawSelfInitCards();
+        dealer.distributeTwoCards();
+
+        outputView.printDistributeCardsMessage(getPlayerNames(dealer.getPlayers()));
+
+        Card card = dealer.getCards().getCards().get(0);
+        outputView.printDealerInitCards(card.toString());
+
+        outputView.printPlayersInitCards(getPlayersCards(dealer));
+    }
+
+    private List<String> getPlayerNames(final Players players) {
+        List<String> playerNames = players.getPlayers().stream()
+                .map(Player::getName)
+                .collect(Collectors.toList());
+        return playerNames;
+    }
+
+    private void play(Player player) {
+        boolean proceed = true;
+        while (proceed && !player.isBust()) {
+            proceed = ask(player);
+        }
+    }
+
+    private boolean ask(Player player) {
+        String answer = inputView.askReceiveMoreCard(player.getName());
+
+        if (answer.equals("n")) {
+            outputView.printCurrentCards(player.getName(), getCurrentCards(player.getCards()));
+            return false;
+        }
+        dealer.giveOneMoreCard(player);
+        outputView.printCurrentCards(player.getName(), getCurrentCards(player.getCards()));
+        return true;
+    }
+
+    private void turnOfDealer() {
+        while (dealer.canDraw()) {
+            dealer.drawOneMoreCard();
+            outputView.printDealerDrawOneMoreCard();
+        }
+    }
+
+    private void finishGame() {
+        List<Player> players = dealer.getPlayers().getPlayers();
+        List<Integer> scores = getPlayersScore(players);
+
+        outputView.printDealerFinalCards(getCurrentCards(dealer.getCards()), dealer.calculateTotalScore());
+        outputView.printPlayerFinalCards(getPlayersCards(dealer), scores);
+
+        Map<Player, Result> playerResultMap = dealer.decideResult();
+        List<Integer> dealerResult = dealer.decideSelfResult();
+
+        Map<String, String> playerResult = getPlayerResult(playerResultMap);
+
+        outputView.printGameResult(dealerResult, playerResult);
+    }
+
+    private List<Integer> getPlayersScore(final List<Player> players) {
+        List<Integer> scores = players.stream()
+                .map(Player::calculateTotalScore)
+                .collect(Collectors.toList());
+        return scores;
```

Comment:
여기도 즉시 반환이 가능해보이네요 :) 
```suggestion
       return players.stream()
                .map(Player::calculateTotalScore)
                .collect(Collectors.toList());
```